client/reader/following/main.jsx

```diff
@@ -96,12 +96,12 @@ const FollowingStream = ( props ) => {
 			<SectionHeader label={ translate( 'Followed Sites' ) }>
 				{ config.isEnabled( 'reader/seen-posts' ) && ! props.hasUnseen && (
 					<Button compact onClick={ markAllAsUnSeen }>
-						{ translate( 'Mark All as Unseen' ) }
+						{ translate( 'Mark All as unseen' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.